src/material-moment-adapter/adapter/moment-date-adapter.spec.ts

```diff
@@ -8,7 +8,8 @@
 
 import {LOCALE_ID} from '@angular/core';
 import {async, inject, TestBed} from '@angular/core/testing';
-import {DateAdapter, DEC, FEB, JAN, MAR, MAT_DATE_LOCALE} from '@angular/material/core';
+import {DateAdapter, DATE_LOCALE} from '@angular/cdk/dateadapter';
+import { DEC, FEB, JAN, MAR } from '@angular/cdk/testing';
```

Comment:
Omit spaces in braces

Reply:
@jelbourn Does it worth to have a tslint rule that enforces consistent spaces? Or maybe better, have you ever considered to use [prettier](https://github.com/prettier/prettier)?

Reply:
All of Google (and Angular core) actually uses clang-format for JS/TS internally, so if we were to do anything it would probably be that. At the beginning of this project we decided not to use it because it wrecks havoc on unit test indenting, and I'm not sure that's really any better than it was in 2016.

Reply:
And what about a rule? It can surely reduce the noise on PRs over whitespaces.

Reply:
Probably, yeah. Feel free to send a PR if you're interested!